packages/admin-ui-react/src/system/create-system.tsx

```diff
@@ -9,59 +16,47 @@ import 'focus-visible/dist/focus-visible'
 import { IconProvider } from './icons'
 import { SystemContext } from './context'
 
-export function createSystem(
-  spec: CreateSystemOptions = {}
-): CreateSystemReturn {
+interface ThemeProviderProps {
+  children?: ReactNode
+  experimentalTheme?: any
+  experimentalDisabledGlobalStyles?: boolean
+}
+
+export function ThemeProvider(props: ThemeProviderProps) {
   const {
+    children,
     experimentalTheme = theme,
     experimentalDisabledGlobalStyles = false,
-  } = spec
+  } = props
 
   const csx = createCsx(experimentalTheme)
```

Comment:
This will happen in runtime for everybody, right?

Reply:
Do you intend to resolve this on another pull request? (for example, exporting the csx already created from the core)